framework/molecules/breadcrumbs/breadcrumbs.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  title: 'Breadcrumbs',
+  label: 'Breadcrumbs',
+  status: 'ready',
+  variants: [{
```

Comment:
Maybe add the default variant here (even if it's not useful). It would be less magical, more predictive (but it's not mandatory)